mne/utils.py

```diff
@@ -2370,6 +2370,30 @@ def _get_fast_dot():
     return fast_dot
 
 
+def _get_reduction(reduction, axis=-1):
+    if isinstance(reduction, string_types):
+        if reduction == 'mean':
+            return lambda x: np.mean(x, axis=axis)
+        elif reduction == 'median':
+            return lambda x: np.median(x, axis=axis)
+        else:
+            raise ValueError('reduction, if string, must be "mean" or "median"'
+                             ', got {}'.format(reduction))
+    elif isinstance(reduction, (float, np.float64, np.float32)):
+        if reduction < 0 or reduction >= 0.5:
+            raise ValueError('reduction, if float, means proportion to trim in'
+                             ' trimmed mean, which has to be > 0 and < 0.5, '
+                             'got {}'.format(reduction))
+        from .fixes import get_trim_mean
```

Comment:
do you really need to nest this? `fixes` shouldn't require anything from `utils`...